src/parsing/logutils.ts

```diff
@@ -24,5 +24,6 @@ export const ambiguate = (nameRealm: string): string[] => {
   const split = nameRealm.split('-');
   const name = split[0];
   const realm = split[1];
-  return [name, realm];
+  const region = split[3];
```

Comment:
I don't see how this works? As far as I understand it the combat log doesn't provide the region. 

For example:  `Alexsmite-TwistingNether`  will end up giving you undefined as the region here.

Reply:
OK - I was wrong - I can see this works now.

But this is shared code and I expect classic logs will not have the region.

It would also I think be nice to parse this into a set of known enums rather than strings. 

I expect eventually it will be nice to map "EU" -> "eu" etc to facilitate linking to WCL / RIO / whatever other things.

Reply:
Agree handling this elsewhere is sensible and fine to leave as a string here. Fits with the raw type. 

I'm nervous about how the realms will be handled. Maintaining a list of all realms, slugs and translations seems unrealistic. But we should discuss that elsewhere.